lib/evm/operation/stack_memory_storage_and_flow.ex

```diff
@@ -91,26 +84,21 @@ defmodule EVM.Operation.StackMemoryStorageAndFlow do
       iex> db = MerklePatriciaTree.Test.random_ets_db()
       iex> state = EVM.Operation.StackMemoryStorageAndFlow.sstore([0x11223344556677889900, 0x111222333444555], %{state: MerklePatriciaTree.Trie.new(db)})[:state]
       iex> EVM.Operation.StackMemoryStorageAndFlow.sload([0x11223344556677889900], %{state: state, stack: []})
-      %{
-        stack: [0x111222333444555]
-      }
+      0x111222333444555
 
       iex> db = MerklePatriciaTree.Test.random_ets_db()
       iex> state = EVM.Operation.StackMemoryStorageAndFlow.sstore([0x11223344556677889900, 0x111222333444555], %{state: MerklePatriciaTree.Trie.new(db)})[:state]
       iex> EVM.Operation.StackMemoryStorageAndFlow.sload([0x1234], %{state: state, stack: []})
-      %{
-        stack: [0x0]
-      }
+      0x0
   """
   @spec sload(Operation.stack_args, Operation.vm_map) :: Operation.op_result
   def sload([key], %{state: state=%Trie{}, stack: stack}) when is_list(stack) do
     # TODO: Consider key value encodings
-    stack_value = case Trie.get(state, <<key::size(256)>>) do
-      nil -> 0
-      value -> :binary.decode_unsigned(value)
+    if value = Trie.get(state, <<key::size(256)>>) do
```

Comment:
This confused me for a moment-- didn't get it was a match/bind and not an equality check